frontend/src/components/Auth/AuthNavbar.vue

```diff
@@ -35,7 +35,8 @@ export default {
   mixins: [authLinks],
   data() {
     return {
-      logo: '/img/brand/green.png',
```

Comment:
delete this file?

Reply:
this must remain because of the exact dimensions